tests/lax_control_flow_test.py

```diff
@@ -1969,7 +1969,7 @@ def loop(loop_impl, x):
     jtu.check_grads(loop_lax, (x,), order=2, modes=["fwd"])
 
   def testStaticForiGrad(self):
-    func = lambda x: lax.fori_loop(x, x + 2., lambda i, c: c, x)
+    func = lambda x: lax.fori_loop(int(x), int(x) + 2, lambda i, c: c, x)
```

Comment:
I don't think this test change is necessary once you address the comments within the `fori_loop` implementation.